### PR TITLE
Docs(README.md): added `Example` section

### PR DESCRIPTION
**Pull request by rowhitswami**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,14 @@ First, add a script for it to `package.json`:
 
 Then run `npm run tslint-check`.
 
+### Example
```
Comment (alexjoverm) [inline README.md]: Let's better name it "Tutorials". Sorry about that

File: README.md [hunk 2]
```diff
@@ -73,6 +73,14 @@ First, add a script for it to `package.json`:
...
+
+Wonder how [TSlint](https://palantir.github.io/tslint/) and [Prettier](https://github.com/prettier/prettier) works together?
```
Comment (alexjoverm) [inline README.md]: Let's leave this out. Just the links are ok

File: README.md [hunk 3]
```diff
@@ -73,6 +73,14 @@ First, add a script for it to `package.json`:
...
+Visit these links to learn more:
+
+- [Using TSlint with Prettier](https://alexjoverm.github.io/2017/06/12/Use-Prettier-with-TSLint-and-be-happy/)
+- [Egghead.io (Video)](https://egghead.io/lessons/typescript-use-prettier-with-tslint-without-conflicts-c39670eb/)
```
Comment (alexjoverm) [inline README.md]: Better the title "Use Prettier with TSLint without conflicts (video)"

**Comment by rowhitswami** [comment]
@alexjoverm  Please review the changes :blush:

**Comment by alexjoverm** [comment]
Done, thanks!

**Comment by rowhitswami** [comment]
Thank you so much for merging @alexjoverm  :smiley:

**Pull request merged**